src/test/java/step_definitions/salesforceDemo.java

```diff
@@ -52,7 +52,7 @@ public void read_the_test_data_from_Excel(String arg1) {
     @When("^navigate to salesforce$")
     public void navigate_to_the_url() throws Exception {
 
-        KeywordUtil.navigateToUrl("https://testing-f5-dev-ed.develop.my.salesforce.com");
+        KeywordUtil.navigateToUrl("https://testingxperts17-dev-ed.develop.my.salesforce.com/");//neelima SF URL
```

Comment:
Please don't remove the line to navigate to the url, just comment it while testing and while committing just comment your line and uncomment the original one